packages/aws-cdk-lib/aws-ecs/lib/cluster.ts

```diff
@@ -232,10 +249,15 @@ export class Cluster extends Resource implements ICluster {
       this._executeCommandConfiguration = props.executeCommandConfiguration;
     }
 
+    if (props.managedStorageConfiguration) {
+      this._managedStorageConfiguration = props.managedStorageConfiguration;
+    }
+
     this._cfnCluster = new CfnCluster(this, 'Resource', {
       clusterName: this.physicalName,
       clusterSettings,
-      configuration: this._executeCommandConfiguration && this.renderExecuteCommandConfiguration(),
+      configuration: (this._executeCommandConfiguration && this.renderExecuteCommandConfiguration())
+        || (this._managedStorageConfiguration && this.renderManagedStorageConfigurationConfiguration()),
```

Comment:
What if the user specifies both props?
What about changing the `renderExecuteCommandConfiguration()` method to `renderConfiguration`?
```ts
  private renderConfiguration(): CfnCluster.ClusterConfigurationProperty | undefined {
    if (!this._executeCommandConfiguration && !this._managedStorageConfiguration) {
      return undefined;
    }
    return {
      executeCommandConfiguration: this._executeCommandConfiguration && {
        kmsKeyId: this._executeCommandConfiguration.kmsKey?.keyArn,
        logConfiguration: this._executeCommandConfiguration.logConfiguration && this.renderExecuteCommandLogConfiguration(),
        logging: this._executeCommandConfiguration.logging,
      },
      managedStorageConfiguration: this._managedStorageConfiguration && {
        fargateEphemeralStorageKmsKeyId: this._managedStorageConfiguration.fargateEphemeralStorageKmsKey?.keyId,
      },
    };
  }
```

Reply:
ahh good catch, I'll change it :)